src/pages/workspace/withPolicyConnections.tsx

```diff
@@ -24,21 +24,25 @@ type WithPolicyConnectionsProps = WithPolicyProps;
 function withPolicyConnections<TProps extends WithPolicyConnectionsProps>(WrappedComponent: ComponentType<TProps>) {
     function WithPolicyConnections(props: TProps) {
         const {isOffline} = useNetwork();
-        const [hasConnectionsDataBeenFetched, {status}] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_HAS_CONNECTIONS_DATA_BEEN_FETCHED}${props.policy?.id ?? '0'}`, {
+        const [hasConnectionsDataBeenFetched] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_HAS_CONNECTIONS_DATA_BEEN_FETCHED}${props.policy?.id ?? '0'}`, {
             initWithStoredValues: false,
         });
+        const isConnectionDataFetched = useMemo(
```

Comment:
The variable name is a little confusing, if the `areConnectionsEnabled` feature is off, this variable would be true and that's not correct. Can you invert the conditions and use something like `isConnectionDataFetchNeeded` as the variable name?